homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/directives/change-link-color.directive.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Input,
+  Directive,
+  AfterViewInit,
+} from '@angular/core';
+
+@Directive({selector: '[appChangeLinkColor]'})
+export class ChangeLinkColorDirective implements AfterViewInit {
+
+  @Input('appChangeLinkColor') el: HTMLAnchorElement
+
+  ngAfterViewInit() {
+    const protocolNode = this.el.querySelector('.vehicle-list__vehicle-item-protocol')
+    if (protocolNode.innerHTML === 'http') {
+      protocolNode.setAttribute('style', 'color: red');
+      this.el.removeAttribute("href")
```

Comment:
Really liked your solution! :+1: 

But here I have some points to fix:
* Some lines have semicolon `;`, some not. Please, set it to the one standard. I suggest you to add semicolons to all lines;
* Some lines use singlequote `'`, some use doublequote `"`. Please, set it to the one standard. I suggest you to use singlequote `'`